crates/topos-tce-storage/src/index/mod.rs

```diff
@@ -17,8 +17,8 @@ use crate::{
     types::{CertificateSequenceNumber, TargetSourceListColumn, TargetStreamsColumn},
 };
 
-#[allow(unused)]
-pub(crate) struct IndexStore {
+pub struct IndexStore {
+    #[allow(unused)]
     certificate_order: DBColumn<CertificateSequenceNumber, CertificateId>,
```

Comment:
Why do we need this field?

Reply:
For now we don't need it, it was planned to use it for the API queries

Reply:
Why don't you add it when we actually need it?